### ISSUE TITLE
[Bug] How to synchronize two or more graphics with converFromPixel in same echarts.instance?

### ISSUE THREAD

**Issue by gioneves** [apache/echarts#20628]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor?code=MYewdgzgLgBMAWBDATlAygT2gUwLYwF4YAKASkID4YBvAKBjnGhgBMAzASTAEtYizKNeg0aRYAc2QgArgAcIhGAG0A5JJmyAjCoA0MNVLkAmXfvVyAzCoC6AblrCGoMa0RREipXYciYbEMgkADbYsNyKAAy2MOEAPDAWUTEA1MnkdL4MLG6IAHSy0hDwxEqOmcQAsm7wuciIYCwguAIAVDCaEZrkAD4wETplvpXVtfWNzeRtHV0wvf2DIubySsNQNXUNTa0wSxC5IWDiaz191gvWpPa-AL5lyKHSyGCu7lcw1_bC91CPzxkiEBkyGA2AAXEJMn5uMhoOD2FxeGQyrcGB9aGjaAB6TEwCDYBq0EKwApBIIAERyeL4ym8tGczAQKHQ-JYigEBCo_3psGoUJhsGuikZqEwOAmuUBjxBnwYJPJlNC-UKxX-DElwLBylUAA9tHoVNqTPrdjY9LlzWxodAzqjLujPsLmQ0yJ82NIwMAoNxwHAkKgAGJ1XDYNAAd14CAiAi5TFgXqgIQinjKqpEUGw2qg4JUzgAbtgA1JcAAFbja7BBGLPCCIYNV6D1EEkR0AVTxpF0CxCbCz-hBYHTyE7kPTmbQUAwIXBqd8_gHaG4AC9NYkBpCnCAggFswBiIwAFiMAA4LCYFiiRBfaRuXFAQJuvbIk0RU1BkNxxOIC9nENruBBhxEX9_2LEBuAHb8IUyIJwJDCcpygyFQC3ZBdwidCVHPZFhAxG9mEkbgWGfZQUy7bAe2zABWABSQDfDvWRs00IxaLXTJwxYNZs33CJWIWeBsA_eBexUCwaMwm42IYGcYHfcRhKoviRxARj9GYpT2MIrj9B4jTfAEoSRLE2jsIYa9RGYbUAEE_wgYjSl8GSJ1kTUVFzRAgmkbA6MWd8WC4FgM3BCJkSkxC0wwFzs3czzvLChgCP8hogvaUyYHM7kYAwGz_3s0iR0i1yYq8nyEr8gKUpCyT8syZyio8kr4p2crku1cFNDSjLY1xAtuGwOzk0chY6uzCBgDcQdSpeRAqQqtq-ia6zbLm4Kmuy5bWtWhZ8VAQLpwWBh5vmdcsvarDqqGgqov0MaJoLKbsncWbNoWhYlv_Fb2jWnKIE-zQmp2kA9vC3wjqahgMDOyEL1RYQupcFSvR9F8ynjBC0ewY603vIJH3BO8H24J8wsS8FEqxrIFV7JQ5QpJ7QiUCJrGsML3ogcE2YprKfvBdbcrCvF3368FBb6uycJlGBcAwABhP0oAlUIAHlZCRsBiER70wDtW46XlwNaxDcMoEjF0HGxGA5aZBQIAwD14CkHhl1oaWrdQXIvygAAtZAyFycBiBUXAZDxYP81MN0PTVkhZBQWsIHSYQLZltwEClkO8BAfMYGwfMB2EIkYFkMCBy4ABxPzPFjoM9hANg2CpAANPRq_j_366pABNcyLdF_q-irQdEE9LXhG4NgY7j3A9j737WsIAgiAiRPfEL8DAu1RRXfl3I8wLKBAyaUtyyCYheVnz6k2uFuS6gcu_NIRnzIYZOBOAABrBRx5gNZsCrDeYgKGNvAcCP8BLTVkvUL8ZRv7EHXhmGAFAl4wAAGQoP_gg-ItMqZPwlECEE-x8RHHgCvTI28mS5BYP-WOJt4BWRHgHGSDARo3XgCAUMAAVImU01S9X6n9cG01PrwO1OeO0NwcI-BgL3PhChNCDwLMPNWwgKx4hiBPYgrdp6KyFnPABi8iAzFTGveeRByHuz3oWI-ZYKxnx6rov67wb7gTvmACuhFH5M0li_HEctsAfy_hPX-GDN7_hgMA0BwTHoeA2NA3wsCRGIOQWgkJMAsHSFJHTGaDMmZ4KlNgQhhxjggylrLHeVCIA0IQPQtWZ8Do_0KqNNhnDuGCIvi9LmvhonCNamIt4sNYbXEuEAA

### Steps to Reproduce

I have two dot graphics (scatter) and I need to move the mouse (mousemove) over one of them and pass the tooltip effect to the other. That is, when I move the mouse over series[0], the tooltip must also appear in series[1] and vice versa. I tried synchronizing the charts with chartUSe.getZr().on(...) but I couldn't get it to work. Also, **it wouldn't be interesting to use echarts.connect to do this, as I need the charts to be in the same instance**.

### Current Behavior

The tooltip is not transferred to the other charts as it naturally is with `echarts.connect`.

### Expected Behavior

The tooltip needs to work on all the graphics programmed for the tooltip action to take place, so if I hover over the first graphic, the tooltip should appear at the same point (axis) as the second and vice versa. Note that all graphs must have the same scale axis.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
I do not think this is a bug. Two series in a **single** chart have a **single** tooltip activated on _mousemove_ by default.
Event _mousemove_ has precedence over action _showTip_. So once a tooltip is activated in one series, _showTip_ will not work in the other because only one tooltip is allowed.

**Comment by gioneves** [comment]
How can I make two tooltips (or more) appear in the same instance? Is there a hack to make this work? Or would it be more appropriate to recommend a new feature with this possibility in `axisPointer`, something like:

`link: [ { xAxisIndex: [0, 1, ..., 2], multipleTooltips: true } ]`

For behavior similar to `echarts.connect`?

**Comment by helgasoft** [comment]
Changing the tooltip present state is a lost cause IMHO. One would rather focus on [data.label](https://echarts.apache.org/en/option.html#series-scatter.data.label). Multiple labels could be turned on/off with _setOption_, on any series.
